lstchain/reco/r0_to_dl1.py

```diff
@@ -545,9 +542,15 @@ def r0_to_dl1(
                                     event.lst.tel[telescope_id].evt.tib_pps_counter +
                                     event.lst.tel[telescope_id].evt.tib_tenMHz_counter * 10 ** (-7)
                             )
+
                             if event.lst.tel[telescope_id].evt.extdevices_presence & 2:
                                 # UCTS presence flag is OK
                                 ucts_time = event.lst.tel[telescope_id].evt.ucts_timestamp * 1e-9  # secs
+                                if math.isnan(first_valid_ucts):
```

Comment:
Ah I see now. This line is the actual fix, right?

Reply:
THen yes, it should be changed back to None. THen the diff also becomes more obvious.

Reply:
Yes. Actually the way I've done it is kind of confusing with too many if/else to take into account the different possibilities

Reply:
Should I change back the nan's to None's?